Automattic/newspack-ads
# chore(deps): googleads-php-lib from v202102 to v202111

Update Google Ads library from the deprecated v202102 to the latest v202111.

Closes #267 

## How to test

Check out this branch and test common GAM tasks:
 1. Connect using a service account (JSON file)
 2. Create a new ad unit
 3. Edit a created ad unit
 4. Delete an ad unit
 5. Visit a page with ads and confirm they are visible
